data/mods/Aftershock/items/armor/modular_exos/exo_tool.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "id": "exo_repair_kit",
+    "copy-from": "large_repairkit",
+    "type": "TOOL",
+    "name": { "str": "exoskeleton repair kit" },
+    "description": "This is a collection of tools including a high-powered torch for working the titanium frame.",
```

Comment:
This description feels incomplete to me, mind explaining what you were going for?

I feel like this doesn't need to be re-worked that much description wise, generic is good for now, but I'm curious what is the idea for this description?

Reply:
Primarily I wanted to have a tool that could work titanium since the standard welders in game can't. Titanium welding is a whole different level of skill already, so the frames would be gated behind a repair proficiency and at least one specialized tool. This could be just a better welder, but until multifuel is an option welding is still abstracted. The vehicle version of this makes more sense as it could provide a shield-gas filled environment and provide solvents/tools to clean the welding sites.